arlo-client/docs/api_sequence.md

```diff
@@ -764,6 +731,370 @@ happened on the first round, would look like this:
+            isComplete: null,
+            pvalue: null,
+          },
+          id: 'contest-1',
+          results: {},
+          sampleSize: 379,
+          sampleSizeOptions: [
+            { size: 269, type: 'ASN', prob: [1] },
+            { size: 379, prob: 0.8, type: null },
+            { size: 78, prob: null, type: null },
+          ],
+        },
+      ],
+      id: 'round-1',
+      endedAt: null,
+      startedAt: 'Thu, 18 Jul 2019 16:34:07 GMT',
+    },
+  ],
+  name: 'contest name',
+  randomSeed: '123456789',
+  riskLimit: 1,
+  online: true,
+}
+```
+
+GET the ballots list for this audit board (similar to the /ballot-list endpoint
+above, but filtered by Audit Board, and thus each `ballot` object doesn't have
+the `auditBoard` property):
+
+- `GET /election/<electionId>/jurisdiction/<jurisdictionId>/audit-board/<board.id>/round/<roundId>/ballot-list`
+
+- `src/components/AuditFlow/index.tsx` > `AuditFlow` > `getBallots()` > `api()`
+
+  ```
+  {
+    ballots: [
+      {
+        comment: null,
+        position: 1,
+        status: null, // status is null!
+        timesSamples: 4,
+        vote: null,
+        batch: {
+          id: 'batch-1',
+          name: '1',
+          tabulator: null,
+        },
+      },
+      ...
+    ]
+  }
+  ```
+
+  ### Enter Audit Board Member information
+
+  Since the `members` array on the audit board object is empty, the member
+  creation form will be shown. Once filled and submitted, it will POST the form
+  data to the backend.
+
+  - `POST /election/${electionId}/jurisdiction/${jurisdictionId}/audit-board/${boardId}`
+
+  - `src/components/AuditFlow/MemberForm.tsx` > `Formik` > `onSubmit` > `api()`
+
+  ```
+  {
+    name: "Audit Board #1",
+    members: [ // Currently we support exactly two members
+      {
+        name: "Member A",
+        affiliation: "IND",
+      },
+      {
+        name: "Member B",
+        affiliant: ""
+      }
+    ]
+  }
+  ```
+
+  Once this is accepted and the above endpoints are pinged again to update the
+  data.
+
+  GET the updated status:
+
+- `GET /election/{electionId}/audit/status`
+
+- `src/components/AuditFlow/index.tsx` > `AuditFlow` > `getSTatus()` > `api()`
+
+```
+{
+  contests: [
+    {
+      choices: [
+        {
+          id: 'choice-1',
+          name: 'choice one',
+          numVotes: 792,
+        },
+        {
+          id: 'choice-2',
+          name: 'choice two',
+          numVotes: 1325,
+        },
+      ],
+      id: 'contest-1',
+      name: 'contest name',
+      numWinners: 1,
+      totalBallotsCast: '2123',
+      votesAllowed: 1
+    },
+  ],
+  jurisdictions: [
+    {
+      auditBoards: [
+        {
+          id: 'audit-board-1',
+          name: 'Audit Board #1',
+          members: [ // The members array isn't empty anymore!
+            {
+              name: "Member A",
+              affiliation: "IND",
+            },
+            {
+              name: "Member B",
+              affiliant: ""
```

Comment:
Do we care about single vs. double quotes in these? I'd propose we use the `jsonc` language (JSON with comments) for these docs so that they're syntax highlighted correctly. That'd require quoting the keys, but I think having prettier format them automatically would be worth it.

Reply:
@eventualbuddha I switched these to single quotes in here, but we should definitely decide on a prettier standard, yeah.

Reply:
https://github.com/votingworks/arlo/pull/269